frame/balances/e2e-tests/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+use substrate_test_runner::{Node, ChainInfo};
+use pallet_sudo::Call as SudoCall;
+use pallet_balances::Call as BalancesCall;
+use sp_keyring::sr25519::Keyring::{Alice, Bob};
+use sp_runtime::{traits::IdentifyAccount, MultiSigner, MultiAddress, MultiSignature};
+use sp_runtime::AccountId32;
+use sp_runtime::traits::Block as BlockT;
+use sp_runtime::generic::UncheckedExtrinsic;
+use codec::Encode;
+use frame_support::traits::Get;
+
+type Extrinsic<R, S> = UncheckedExtrinsic<
+    MultiAddress<<R as frame_system::Config>::AccountId, <R as frame_system::Config>::Index>,
+    <R as frame_system::Config>::Call,
+    MultiSignature,
+    S,
+>;
+
+/// Tests the Balances::force_transfer call
+pub fn force_transfer<T>(node: &Node<T>)
+    where
+        T: ChainInfo,
+        T::Runtime: pallet_sudo::Config + pallet_balances::Config,
+        <T::Runtime as frame_system::Config>::Call: From<SudoCall<T::Runtime>> + From<BalancesCall<T::Runtime>> + Encode,
+        <T::Runtime as pallet_sudo::Config>::Call: From<BalancesCall<T::Runtime>>,
+        <T::Runtime as pallet_balances::Config>::Balance: From<u8>,
+        <T::Runtime as frame_system::Config>::AccountId: From<AccountId32> + Encode,
+        <T::Block as BlockT>::Extrinsic: From<Extrinsic<T::Runtime, T::SignedExtras>>,
+        <<T::Runtime as frame_system::Config>::Lookup as sp_runtime::traits::StaticLookup>::Source: From<AccountId32>,
+        AccountId32: std::borrow::Borrow<<T::Runtime as frame_system::Config>::AccountId>
+{
+    type Balances<T> = pallet_balances::Module<T>;
+    let (alice, bob) = (
+        MultiSigner::from(Alice.public()).into_account(),
+        MultiSigner::from(Bob.public()).into_account(),
+    );
+    let (alice_balance, bob_balance) = node.with_state(|| (
+        Balances::<T::Runtime>::free_balance(alice.clone()),
+        Balances::<T::Runtime>::free_balance(bob.clone()),
+    ));
+    let balance = alice_balance / 2u8.into();
+
+    let balances_call = BalancesCall::force_transfer(alice.clone().into(), bob.clone().into(), balance.into());
+    node.submit_extrinsic(
+        SudoCall::sudo(Box::new(balances_call.into())),
+        alice.into()
+    );
+    node.seal_blocks(1);
+
+    // TODO: figure out events
```

Comment:
if this is something we need that often (and I think we do), why not add a helper function on the ChainInfo-Trait ... similarly to the signed-extrinsic helper.?

Reply:
This could also work 🤔